boilerplates/react-drizzle/files/pages/todo/+data.ts

```diff
@@ -0,0 +1,10 @@
+import { db } from "@batijs/drizzle/database/db";
+import { todoTable } from "@batijs/drizzle/database/schema";
+
+export type Data = ReturnType<typeof data>;
+
+export default function data() {
+  const todoItems = db.select().from(todoTable).all();
+
+  return todoItems;
+}
```

Comment:
This is also reusable across boilerplates, so move it to `boilerplates/drizzle`. (I'll reference this as **C** in next comments).